lib/client-assets.php

```diff
@@ -177,6 +177,13 @@ function gutenberg_register_scripts_and_styles() {
 		filemtime( gutenberg_dir_path() . 'blocks/build/edit-blocks.css' )
 	);
 	wp_style_add_data( 'wp-edit-blocks', 'rtl', 'replace' );
+
+	if ( WP_DEBUG ) {
```

Comment:
Would `SCRIPT_DEBUG` be the more appropriate constant here? Based on the description, it seems primarily aimed at toggling minified version, so perhaps not.

https://codex.wordpress.org/Debugging_in_WordPress#SCRIPT_DEBUG